README.md

```diff
@@ -110,6 +110,37 @@ const myPlugin = {
 };
 ```
 
+## Custom fields
+
+Puck supports custom fields using the `custom` field type and `render` method.
+
+In this example, we optionally add the `<FieldLabel>` component to add a label:
+
+```tsx
+import { FieldLabel } from "@measured/puck";
```

Comment:
Needs testing. I was getting issues when testing via demo application without reaching directly into the dir (i.e. I had to import via `import { FieldLabel } from "@measured/puck/components/InputOrGroup"`)